src/main/java/edu/harvard/iq/dataverse/api/MakeDataCountApi.java

```diff
@@ -158,7 +150,7 @@ public Response updateCitationsForDataset(@PathParam("id") String id) throws Mal
                     logger.warning("Failed to get citations from " + url.toString());
                     return error(Status.fromStatusCode(status), "Failed to get citations from " + url.toString());
                 }
-                JsonObject report = Json.createReader(connection.getInputStream()).readObject();
+                JsonObject report = JsonUtil.getJsonObject(connection.getInputStream());
```

Comment:
Could/should have a try with resources for this input stream? (And any uses of your new getJsonObjectArray(InputStream) methods.)

Reply:
You're correct. Let me fix that soon. At least I didn't make the situation worse, I think. Thanks for reviewing!